doc/project/ip_checklist.md.tpl

```diff
@@ -11,7 +11,7 @@ All checklist items refer to the content in the [Checklist.]({{< relref "/doc/pr
 
 Type          | Item                  | Resolution  | Note/Collaterals
 --------------|-----------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]     | Not Started |
+Documentation | [SPEC_COMPLETE][]     | Not Started | [${name.upper()} Design Spec]({{<relref "hw/ip/${name}/doc" >}})
```

Comment:
Just one minor nit: it might be neater to move the relref link below the table (just like the entries in the Item column).

Reply:
I would leave it up there as these links have to be adjusted when creating a new IP, i.e., ${name.upper()} and ${name} has to be replaced by the actual module name. I think it gets less likely lost for reviewers this way. In contrast, the other links below the tables are not edited ever.

Reply:
Sounds good, thanks!

Reply:
I'll add it to `uvmdvgen` and get it automated so that those replacements are not needed.

Reply:
Ah, that would be smart solution!